src/TTFFont.js

```diff
@@ -90,6 +90,9 @@ export default class TTFFont {
    * @type {string}
    */
   get postscriptName() {
+    if (this.name === undefined || this.name.records.postscriptName === undefined) {
```

Comment:
note that an OpenType font without a name table is an illegal font: the `name` table is a hard requirement.